### ISSUE TITLE
Example TensorFlow: VGG19  output wrong answer

### ISSUE THREAD

**Issue by VVVXYZ** [bethgelab/foolbox#364]
> bad example

I run example TensorFlow: VGG19 from  https://foolbox.readthedocs.io/en/latest/user/examples.html#tensorflow-vgg19,and I got wrong answer.
Here is the code:

`import tensorflow as tf
from tensorflow.contrib.slim.nets import vgg,resnet_v2
import numpy as np
import foolbox

images = tf.placeholder(tf.float32, shape=(None, 224, 224, 3))
preprocessed = images - [123.68, 116.78, 103.94]
logits, _ = vgg.vgg_19(preprocessed, is_training=False)
#logits, _ = resnet_v2.resnet_v2_50(preprocessed, is_training=False)
#logits = tf.squeeze(logits, axis=[1, 2])
restorer = tf.train.Saver(tf.trainable_variables())

image, trueLabel = foolbox.utils.imagenet_example()

path="E:/workspace/model/slim"
with foolbox.models.TensorFlowModel(images, logits, (0, 255)) as model:
    restorer.restore(model.session, path+'/vgg_19.ckpt')#resnet_v2_50
    print("trueLabel  ",trueLabel )
    print("preLabel  ",np.argmax(model.predictions(image)))
`

Here is the output

`trueLabel   282
preLabel   281`

Is it a bug,or just a bad imge which is classifide as wrong label?

**Comment by jonasrauber** [comment]
I think it's just an image that get's misclassified by this particular model (the ResNet-50 we use in the other examples does classify it correctly).
